### PR TITLE
some linting

### PR DESCRIPTION
**Pull request by fperrad**

### CODE REVIEW

File: src/misc/crypt/crypt_register_all_prngs.c [hunk 1]
```diff
@@ -16,7 +16,7 @@
 */
 
 #define REGISTER_PRNG(h) do {\
-   LTC_ARGCHK((err = register_prng(h) == -1 ? CRYPT_ERROR : CRYPT_OK) == CRYPT_OK); \
+   LTC_ARGCHK((err = register_prng(h)) == -1); \
```
Comment (karel-m) [inline src/misc/crypt/crypt_register_all_prngs.c]: You probably wanted to say `LTC_ARGCHK((err = register_prng(h)) != -1);`

Comment (fperrad) [inline src/misc/crypt/crypt_register_all_prngs.c]: Yes.
Fixed.

**Comment by karel-m** [comment]
👍 @sjaeckel if you find it also OK, please do merge

**Comment by sjaeckel** [comment]
Please rebase and I'll merge ASAP

and please push to this repo and not your fork the next time, then I could do the rebase for you

**Comment by fperrad** [comment]
rebased

**Pull request merged**